tests/resources/utils-data.ts

```diff
@@ -0,0 +1,94 @@
+import { FacetDisplayNames, Taxonomy } from '../../src/SearchIndex';
+
+export const sampleTaxonomy = {
+  genres: [{ name: 'reference' }, { name: 'tutorial' }],
+  target_platforms: [
+    { name: 'atlas', versions: [{ name: 'v1.2' }, { name: 'master' }] },
+    {
+      name: 'atlas-cli',
+      display_name: 'Atlas CLI',
+      versions: [{ name: 'v1.2', stable: true }, { name: 'master' }],
+    },
+    {
+      name: 'manual',
+      versions: [{ name: 'v1.0' }, { name: 'master' }],
+    },
+    {
+      name: 'spark-connector',
+      display_name: 'Spark Connector',
+      versions: [{ name: 'v2.0' }, { name: 'v2.1' }],
+    },
+    { name: 'node', versions: [{ name: 'v4.9' }] },
+    {
+      name: 'mongocli',
+      display_name: 'Mongo CLI',
+      versions: [{ name: 'v1.0' }],
+    },
+    { name: 'visual-studio-extension', versions: [{ name: 'current' }] },
+    { name: 'golang', versions: [{ name: 'v1.7' }] },
+    { name: 'java', versions: [{ name: 'v4.3' }] },
+  ],
+} as Taxonomy;
+
+export const sampleFacetTrie = {
+  genres: {
+    name: 'Genres',
+    reference: { name: 'Reference' },
+    tutorial: { name: 'Tutorial' },
+  },
+  target_platforms: {
```

Comment:
Just curious if we're intentionally leaving `target_platforms` as snake case, or does it not matter for this field?

Reply:
this was intentionally left as snake case to coincide with the original taxonomy.toml. snake case should be consistent across the chain of process along taxonomy -> search index -> aggregation queries -> query parameters